src/main/java/cz/centrum/haffner/SimpleTrainingDavid/AppServices/OneDayFileJsonParser.java

```diff
@@ -139,25 +161,59 @@ public MetricsInfoData process(File inputFile) {
                 if (destinationCountryCode >0) {                            // value of -1 or 0 means invalid code
                     kpisInfoData.addDifferentDestinationCodesSet(destinationCountryCode);}
 
-                // TODO (final delete): sleeping block for debugging proposes only
+                // sleeping block for higher visibility of particular processes duration
                 try {
                     Thread.sleep(50);
                 } catch(InterruptedException ex){
-                    // do stuff
+                    logger.error(ex.getMessage(), ex);
                 }
 
                 Instant endingJsonProcess = Instant.now();
                 kpisInfoData.addJsonProcessingDuration(
                         Duration.between(startingJsonProcess, endingJsonProcess).toMillis() );
+
+                rowsCounter++;
+                if(logger.isDebugEnabled()) {
+                    logger.debug("Successfully finished processing of row no: " + rowsCounter);
```

Comment:
Používej prosím tento zápis:
logger.debug("Successfully finished processing of row no: {}", rowsCounter);

Reply:
Předěláno.